default.xml

```diff
@@ -35,11 +35,35 @@
            sync-j="4"
 	   sync-f="true" />
 
-<!-- BEGIN CYANOGENMOD -->
+<!-- BEGIN RR -->
   <project path="build" name="android_build" groups="pdk" remote="rr" >
     <copyfile src="core/root.mk" dest="Makefile" />
   </project>
   <project path="android" name="platform_manifest" remote="rr"/>
+  <project path="external/android-visualizer" name="android_external_android-visualizer" remote="rr"/>
+  <project path="external/DUtils" name="external_DUtils" remote="rr" />
+  <project path="external/sepolicy" name="android_external_sepolicy" groups="pdk" remote="rr" />
+  <project path="frameworks/av" name="frameworks_av" groups="pdk" remote="rr" />
+  <project path="frameworks/base" name="android_frameworks_base" groups="pdk-cw-fs,pdk-fs" remote="rr" />
+  <project path="frameworks/opt/cards" name="ResurrectionRemix/frameworks_opt_cards" remote="github" revision="marshmallow" />
```

Comment:
I think `project path="frameworks/opt/cards" name="frameworks_opt_cards" remote="rr"` is smarter